packages/terra-grid/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "terra-grid",
+  "main": "lib/grid.js",
```

Comment:
I just notice that. We have the capital filename in lib: `lib/Grid.js`. In `package.json`, we have lower case. It may work, but do we want to keep it consistent?